package.json

```diff
@@ -15390,7 +15399,7 @@
 				},
 				{
 					"command": "gitlens.createCloudPatch",
```

Comment:
Should be the new `gitlens.graph.createCloudPatch` command

Reply:
added